lib/features/sign_in/QA.md

```
@@ -0,0 +1,165 @@
+---
+
+# sign_in QA
+
+## 1. サインインシートの表示（起動元ごとの文言差分）
+
+- [x] **初期設定からの表示**: 初期設定1/3画面で「すでにアカウントをお持ちの方はこちら」をタップすると下からシートが表示され、タイトルが「ログイン」、ボタンの文言が「〜でサインイン」になる
+- [x] **記録画面からの表示**: 記録画面のアカウント登録案内バーをタップすると、タイトルが「アカウント登録」のシートが表示され、ボタンの文言が「〜で登録」になる
+- [x] **設定画面からの表示**: 設定画面のアカウント設定で未連携の項目の「連携する」をタップすると、タイトル「アカウント登録」のシートが表示される
+- [x] **共通UI要素**: シート上部にドラッグバー、タイトル、説明文、Apple/Googleの2つのボタンが表示される
+
+#### 動作確認
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)